### PR TITLE
Disable pip dev preview flag

### PR DESCRIPTION
**Pull request by nirzari**

### CODE REVIEW

File: cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py
Comment (chmeliik) [inline cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py:36]: Shouldn't the downgrade re-enable the flag?

File: tests/integration/test_pip_packages.py
Comment (chmeliik) [inline tests/integration/test_pip_packages.py:31]:
```suggestion
        payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": ["pip"]}
```
`black` is complaining about this

Comment (nirzari) [reply tests/integration/test_pip_packages.py]: Yes I noticed just now. fixed it

File: cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py
Comment (nirzari) [reply cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py:36]: Should it? :thinking: I thought the idea was to make sure that we don't re-enable it in anyway and hence, we could just leave the downgrade blank

**Review by akhmanova — COMMENTED** [review]
Everything related to the integration test looks good to me! Nice work. Let wait for approval related other parts :)

File: cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py
Comment (chmeliik) [inline cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py:45]: This change is in the wrong commit (should be in "Migration for disabling pip-dev-preview flag")

Comment (athos-ribeiro) [reply cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py:45]: In this matter, the commit order seem to be inverted as well: We should first remove the filters for the flag and then disable it in the DB.

**Comment by athos-ribeiro** [comment]
Nice work! LGTM, there is just a couple comments on commit order and on which contents should go where (Adam's comment).

File: cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py
Comment (nirzari) [reply cachito/web/migrations/versions/f133002ffdb4_disable_pip_dev_review.py:45]: Good catch, Fixed it

**Comment by akhmanova** [comment]
Sorry for making more work for you, could you please delete the flag in test_pip_local_path as well https://github.com/release-engineering/cachito/pull/291 ?

**Review by athos-ribeiro — APPROVED** [review]

**Pull request merged**
